### PR TITLE
Update 'Components included in the app' with examples

### PR DESCRIPTION
**Pull request by Farzannajipour**
Fix comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
  
  ### Components included in the app
   - Auth module: define authentication
-  - Locale selector for SPA: support multiple languages
-  - Theme: integrate with a theme
-  - Entitlements: configure entitlements for different scenarios
+  - Locale selector for SPA: support multiple languages. Check the example codes [here](https://github.com/Backbase/golden-sample-app/tree/master/apps/golden-sample-app/src/app/locale-selector)
+  - Entitlements: configure [entitlements](https://community.backbase.com/documentation/foundation_angular/latest/entitlements) for different scenarios. Check the example code in the [`app.component.html`](https://github.com/Backbase/golden-sample-app/blob/738df3de9fb15f96c9fa2017d00269ced63bebaa/apps/golden-sample-app/src/app/app.component.html#L50) and [`entitlementsTriplets.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/services/entitlementsTriplets.ts)
   - Configure journeys
+    - This is [an example](https://github.com/Backbase/golden-sample-app/blob/master/libs/transactions/src/lib/services/transactions-journey-config.service.ts) is how the journey configuration might be defined in the application
```
Comment (tcorral) [inline README.md:38]: There is an extra `is` that can be replaced by 'of'.  "This is an example **is** how" => "This is an example of how"

File: README.md [hunk 2]
```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
...
   - Configure journeys
+    - This is [an example](https://github.com/Backbase/golden-sample-app/blob/master/libs/transactions/src/lib/services/transactions-journey-config.service.ts) is how the journey configuration might be defined in the application 
+    - By default, it will provide some value intended by the journey developers and exported by public API. Check the [`index.ts`](https://github.com/Backbase/golden-sample-app/blob/master/libs/transactions/src/index.ts#L1), However, you can customize values during the creation of applications like [`transactions-journey-bundle.module.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/transactions/transactions-journey-bundle.module.ts#L15)
```
Comment (tcorral) [inline README.md:39]: "However" in capital case after a comma is not correct, please fix it.

File: README.md [hunk 3]
```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
...
   - Communication service: communications between journeys
```
Comment (tcorral) [inline README.md:40]: What about "- Communication service or how to communicate between journeys"

File: README.md [hunk 4]
```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
...
-  - Simple examples of journeys such as transactions and transfer
+    - The general explanation of the communication service idea and theoretical underlying can be found in [communicate_between_journeys](https://community.backbase.com/documentation/foundation_angular/latest/communicate_between_journeys)
+    - There are 3 parts of the communication chain:
+      - Source journey (the one that will send some data or signal) should define and export. Check the [`make-transfer-communication.service.ts`](https://github.com/Backbase/golden-sample-app/blob/master/libs/transfer/src/lib/services/make-transfer-communication.service.ts)
+      - the destination journey (the one that will receive the data of signal) should also define the interface of the service that it expects. Check the [`communication/index.ts`](https://github.com/Backbase/golden-sample-app/blob/master/libs/transactions/src/lib/communication/index.ts)
```
Comment (tcorral) [inline README.md:44]: Capital case, please.

"- Destination" (remove the 'the')

File: README.md [hunk 5]
```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
...
+      - the actual implementation of the service will live on the application level and must implement both of the available interfaces (or abstract classes) from source and destination journeys. Check the example [`journey-communication.service.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/services/journey-communication.service.ts)
```
Comment (tcorral) [inline README.md:45]: Capital case

Comment (tcorral) [inline README.md:45]: "will live" => "lives"

File: README.md [hunk 6]
```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
...
+      - Do not forget, that communication service from the application level should be provided to the journeys modules in the bundle files (to avoid breaking lazy loading). Check the [`transactions-journey-bundle.module.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/transactions/transactions-journey-bundle.module.ts)
```
Comment (tcorral) [inline README.md:46]: This is below "There are 3 parts of the communication chain" but it should be a level up.

**Comment by yenobi** [comment]
@Farzannajipour please, fix the comments

**Comment by santam85** [comment]
@Farzannajipour what shall we do with this PR? It's getting stale...

**Comment by Farzannajipour** [comment]
> 
I will resolve the comments and will merge it

File: README.md [hunk 7]
```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
...
+      - The actual implementation of the service lives on the application level and must implement both of the available interfaces (or abstract classes) from source and destination journeys. Check the example [`journey-communication.service.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/services/journey-communication.service.ts)
+      - Do not forget, that communication service from the application level should be provided to the journeys modules in the bundle files (to avoid breaking lazy loading). Check the [`transactions-journey-bundle.module.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/transactions/transactions-journey-bundle.module.ts)
+    - The general explanation of the communication service idea and  its theoretical underlying can be found in [communicate_between_journeys](https://community.backbase.com/documentation/foundation_angular/latest/communicate_between_journeys)
```
Comment (kirillsud) [inline README.md:47]:
```suggestion
    - The general explanation of the communication service idea and  its theoretical underlying can be found in [Understand communication between journeys](https://community.backbase.com/documentation/foundation_angular/latest/communicate_between_journeys)
```

**Review by kirillsud — CHANGES_REQUESTED** [review]
Good job!

File: README.md (hunk 7 above)
Comment (kirillsud) [reply README.md:47]: I suppose it's better to use the page title here

Comment (Farzannajipour) [reply README.md:47]: That's a good suggestion

**Review by kirillsud — APPROVED** [review]

**Pull request merged**